build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.1'
+        classpath 'com.android.tools.build:gradle:3.4.1'
```

Comment:
This has caused some build issues, can we change it back?

Reply:
I don't have any reason to not change it back

Reply:
The way I tested it was to blur images using the gaussian filter I put into ImageUtiles and then measure the metric compared to non blurred images, the result being empirically tested to be metric of 950.0

Reply:
![image](https://user-images.githubusercontent.com/19329250/59161801-0d65cd80-8ab5-11e9-99e2-c1d6e6851b8a.png)
Current UI without trash can